CONTRIBUTING.md

```diff
@@ -57,8 +57,25 @@ A PR is considered to be **ready to merge** when:
   people reasonable time to review.
 * Trivial changes (typo, cosmetic, doc, etc.) do not have to wait for
   one day and may be merged with a single Maintainer's approval.
-* `CHANGELOG.md` has been updated to reflect what has been
-  added, changed, removed, or fixed.
+* `CHANGELOG.md` file in this repo is autogenerated from `.yaml` files in the 
+  `./.chloggen` directory.
+
+  Your pull-request should add a new .yaml file to this directory. The name of 
+  your file must be unique since the last release.
+
+  During the collector release process, all `./chloggen/*.yaml` files are 
```

Comment:
This one is not only used in the Collector so let's make it a bit more general :) 
```suggestion
  During the release process, all `./chloggen/*.yaml` files are 
```

Reply:
Good point thanks